.gitlab-ci.yml

```diff
@@ -83,6 +88,21 @@ build-in-docker-static:
   variables:
     EXTRA_CMAKE_FLAGS: "-DBUILD_SHARED_LIBS=OFF"
 
+upload-artifacts:
+  stage: upload
+  extends: .in-prplmesh-builder
+  script:
+    - ln -s "$CI_OWNCLOUD_PASSWORD" ~/.netrc
+    - sed -i 's/^/machine ftp.essensium.com login prplmesh-robot-ci password /' ~/.netrc
+    - ci/upload_to_owncloud.sh -v --overwrite-remote-path "artifacts/$CI_JOB_ID" artifacts/latest ./build
+  rules:
+    - if: $CI_COMMIT_BRANCH == "master"
```

Comment:
Perhaps we should keep the artifacts in owncloud even for non-master builds, and in addition overwrite `artifacts/latest for master builds?

Reply:
The annoying thing here is that we have to do a move at the end to make sure the "latest" folder always points to something valid, which means we won't keep older artifacts for the master branch (ideally we would use and update a symlink, but from a quick look it doesn't seem possible with webdav).
We could upload and keep the artifacts for non-master branches regardless, but I found it weird to keep those only and not the ones from master.

Reply:
I would upload everything to a folder with the commit hash in its name, and just upload a second time to latest only for master.

Reply:
That's the other option, but I couldn't convince myself to upload the same things twice in a row... :upside_down_face: 
Actually it's not mentioned anywhere in the owncloud documentation, but webdav has a COPY method, so if owncloud accepts it we have a solution :-) 
http://webdav.org/specs/rfc4918.html#METHOD_COPY

Reply:
For webdav, a symlink looks roughly like this:
latest.html:
```
<html>
<head>
<meta http-equiv="refresh" content="2;url=http://example.com/ACTUAL_LATEST_DIRECTORY" />
<title>Latest version of test results</title>
</head>
<body>
Click <a href="http://example.com/ACTUAL_LATEST_DIRECTORY">here</a> to go to the latest results, if your web browser does not redirect you automatically.
</body>
</html>
```

Actually, using a `.htaccess` file would be even better, but I don't know if that's an option with owncloud